src/test/scala/com/databricks/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -115,6 +118,26 @@ class RedshiftSourceSuite
       new BucketLifecycleConfiguration().withRules(
         new Rule().withPrefix("").withStatus(BucketLifecycleConfiguration.ENABLED)
       ))
+    val mockManifest = Mockito.mock(classOf[S3Object], Mockito.RETURNS_SMART_NULLS)
```

Comment:
This messy mocking is only to get the unit tests to pass. It's not the most high-fidelity mocking in the world, but that's okay because this code path is well-exercised by the integration tests.